File: clean.py
import os
from glob import glob


def remove_files(s):
    def remove_file(path: str):
        if os.path.exists(path):
            os.remove(path)

    files = glob(s)
    for file in files:
        remove_file(file)


remove_files("*.exe")
remove_files("*.out")
remove_files("*.pdb")
remove_files("*.tds")
remove_files("*.obj")

File: clean.pl
#!/usr/bin/env perl
use strict;
use warnings;

sub remove_files {
    my ($pattern) = @_;
    for my $file (glob $pattern) {
        unlink $file if -e $file;
    }
}

remove_files("*.exe");
remove_files("*.out");
remove_files("*.pdb");
remove_files("*.tds");
remove_files("*.obj");
